### ISSUE TITLE
after firmware flashing bootloader mode stands

### ISSUE THREAD

**Issue by TheIsotopes** [betaflight/betaflight-configurator#4156]
### Describe the bug

on the latest master the bootloader mode is not exited after flashing a firmware.

i have this problem with f4 and h7 boards.

### To Reproduce

flashing a firmware and wait on reboot ... but it stays in bootloader mode

### Expected behavior

reset to setup mode after flashing

### Configurator version

latest PWA master snapshot

**Comment by haslinghuis** [comment]
Did you give permission to the port before during normal connection ?

**Comment by haslinghuis** [comment]
![image](https://github.com/user-attachments/assets/5ee0e08a-650b-47c1-a47c-16078bf00685)

**Comment by TheIsotopes** [comment]
here my screenshot after flashing: the entry `USB device successfully closed` is missing in my log
![Screenshot 2024-09-05 192840](https://github.com/user-attachments/assets/241a68e3-d165-4fbf-bfe7-cbec0a47d1d6)

what you mean with with 

> Did you give permission to the port before during normal connection ?

**Comment by TheIsotopes** [comment]
and here my console output
![image](https://github.com/user-attachments/assets/6f429363-c784-49b8-b7dc-c9a3bf095c3c)

**Comment by haslinghuis** [comment]
![image](https://github.com/user-attachments/assets/0c46e852-bf35-4567-be39-681a91b0aa10)

Some processes now give a notification when finished when you gave it permission.

**Comment by TheIsotopes** [comment]
As you can see in the screenshot above, the port is not closed after flashing.

**Comment by haslinghuis** [comment]
Even with your settings it works for me on Ubuntu 22.04.4LTS with chromium. What OS / browser is used ?

![image](https://github.com/user-attachments/assets/63f22aae-d648-4498-be25-7dcaa7ac22d3)

**Comment by TheIsotopes** [comment]
I'm using Microsoft Edge on Windows 11 23H2

**Comment by TheIsotopes** [comment]
Ok, there seems to be a problem with Microsoft Edge, I just installed Chrome and it works without any problems.

**Comment by TheIsotopes** [comment]
@haslinghuis pls can you reopen this issues ... on latest google chrome now i have the same problem

![image](https://github.com/user-attachments/assets/eb4f8a7d-9745-48f5-aa78-0f68377fd9e8)

**Comment by haslinghuis** [comment]
@TheIsotopes - Did you accept permission for notifications introduced in [4134](https://github.com/betaflight/betaflight-configurator/commit/217373f085b3017f982e2e8aef48a18fad1f1ed2) ?

**Comment by TheIsotopes** [comment]
nope ... was deactivated, activated again and it's working now.

I don't think this is how it should be!?

**Comment by haslinghuis** [comment]
Where you on the onboard logging tab (blackbox) or firmware flasher tab ?

**Comment by TheIsotopes** [comment]
In both console screenshot i was on firmware flasher tab.

**Comment by TheIsotopes** [comment]
@haslinghuis tested latest build, but it's not fixed (tested on edge) ... activated again the notification and it's working

![image](https://github.com/user-attachments/assets/4c5d01e3-174b-4b02-8e03-342f063b46cf)

**Comment by TheIsotopes** [comment]
configurator was updated after request

![image](https://github.com/user-attachments/assets/472dede5-fc26-41e9-99f2-c906bc50de1d)

**Comment by haslinghuis** [comment]
Yes was not ready yet and have it working now in a new PR.

But there is one issue if user blocked the first request:
As it looks now browser stores this decision internal and won't be requested after refresh. Only reset browser settings will trigger the request again.

![Screenshot from 2024-09-20 14-20-48](https://github.com/user-attachments/assets/fea39d54-ee99-4d3b-96bd-39025d62e151)

And then enabling 

![image](https://github.com/user-attachments/assets/3b3d6bc9-beeb-4ee3-86a8-d71996511d52)

permissions are still denied and will throw the error.

**Comment by haslinghuis** [comment]
With latest iteration users will not be able to enable the checkbox without giving permission.
